Packages/org.plunderludics.UnityHawk/Runtime/Emulator/Emulator.cs

```diff
@@ -538,24 +559,13 @@ public void OnEnable()
             _emuhawk.ErrorDataReceived += new DataReceivedEventHandler((sender, e) => LogBizHawk(sender, e, true));
         }
 
-        Debug.Log($"{exePath} {string.Join(' ', args)}");
+        Debug.Log($"[unity-hawk] {exePath} {string.Join(' ', args)}");
```

Comment:
Pretty minor but i think if we do this here we should do it everywhere?

Reply:
yeah, i will do as i find them